orte/mca/ess/base/ess_base_std_orted.c

```diff
@@ -611,13 +625,22 @@ int orte_ess_base_orted_setup(char **hosts)
 
 int orte_ess_base_orted_finalize(void)
 {
+    orte_ess_base_signal_t *sig;
+    unsigned int i;
+
     if (signals_set) {
-        /* Release all local signal handlers */
         opal_event_del(&epipe_handler);
         opal_event_del(&term_handler);
         opal_event_del(&int_handler);
-        opal_event_signal_del(&sigusr1_handler);
-        opal_event_signal_del(&sigusr2_handler);
+        /** Remove the USR signal handlers */
```

Comment:
This is a nit pick, but: I don't think this comment matches the code.  The loop below doesn't remove (just) SIGUSR1 and SIGUSR2 like the comment implies.  This comment should likely be fixed or removed (at least on master -- here on the release branch, it's probably not as important) so as not to cause confusion in the future.